DependencyInjection/DoctrineExtension.php

```diff
@@ -224,6 +227,10 @@ protected function loadDbalConnection($name, array $connection, ContainerBuilder
                 $connection['mapping_types'],
             ]);
 
+        $container
+            ->registerAliasForArgument($connectionId, Connection::class, sprintf('%sConnection', $name))
```

Comment:
is `registerAliasForArgument` available in all supported Symfony versions ? I don't remember when the shortcut was added

Reply:
The minimum version for `symfony/dependency-injection` on the `2.3.x` branch is `4.3.3` [which does support this](https://github.com/symfony/symfony/blob/dd4d355126184444fb3f940d17f83a9462f15be9/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L1375).